packages/preferences/src/browser/views/preference-editor-widget.ts

```diff
@@ -279,6 +279,8 @@ export class PreferencesEditorWidget extends BaseWidget implements StatefulWidge
                     const { id, collection } = this.analyzeIDAndGetRendererGroup(candidate.id);
                     const renderer = collection.get(id);
                     if (renderer?.visible) {
+                        // When filtered, treat the first visible child as the selected node, since it will be the one scrolled to.
+                        this.lastUserSelection = renderer.nodeId;
```

Comment:
Fixes the problem with opening multiple nodes when filtering.

Reply:
Looks good 👌